pkg/controllers/metrics/pod/controller.go

```diff
@@ -71,18 +72,47 @@ var (
 			Objectives: metrics.SummaryObjectives(),
 		},
 	)
+	podBoundDurationSeconds = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "karpenter",
+			Subsystem: metrics.PodSubsystem,
+			Name:      "bound_duration_seconds",
+			Help:      "The time from pod creation until the pod is bound.",
+			Buckets:   metrics.DurationBuckets(),
+		},
+		labelNames(),
```

Comment:
Since this is a histogram and historical, we shouldn't differentiate the buckets by this many labels -- we just want an aggregated view historically of all of the pod bound times -- I actually don't think we need any _labels_ here